modules/mkdocs.py

```diff
@@ -0,0 +1,166 @@
+from modules.common.module import BotModule
+import requests
+import re
+
+class MatrixModule(BotModule):
+    def __init__(self, name):
+        super().__init__(name)
+        self.rooms = dict()
+        self.sites = dict()
+        self.site_data = dict()
+
+    def set_settings(self, data):
+        super().set_settings(data)
+        if data.get('rooms'):
+            self.rooms = data['rooms']
+        if data.get('sites'):
+            self.shows = data['sites']
+
+    def get_settings(self):
+        data = super().get_settings()
+        data['rooms'] = self.rooms
+        data['sites'] = self.sites
+        return data
+
+
+    def matrix_start(self, bot):
+        super().matrix_start(bot)
+        self.bot = bot
+        self.add_module_aliases(bot, ['search','find'])
+
+    async def matrix_message(self, bot, room, event):
+        args = event.body.split()
+        cmd = args.pop(0).lower()
+        if cmd == f'!{self.name}' and len(args):
+            cmd = args.pop(0).lower()
+        if cmd[0] == '!':
+            cmd = cmd[1:]
+        if cmd in ['setup']:
+            bot.must_be_admin(room, event)
+            self.logger.info(f"Setting up search for room {room.name}")
+            self.rooms[room.room_id] = {
+                'sites': {}
+            }
+            bot.save_settings()
+            sites = self.rooms[room.room_id]
+            return
+        sites = self.rooms[room.room_id]
+
+        if cmd in ['help']:
+            return await bot.send_text(room, event, self.long_help(bot=bot, room=room, event=event))
```

Comment:
Swap the help with the `room.room_id`, otherwise running `!mkdocs help` in an unconfigured room will KeyError out.